plugin/src/_rules/border.js

```diff
@@ -0,0 +1,59 @@
+import { handler as h, directionMap, cornerMap } from "#utils";
+
+const borderStyles = [
+  "solid",
+  "dashed",
+  "dotted",
+  "double",
+  "hidden",
+  "none",
+  "groove",
+  "ridge",
+  "inset",
+  "outset",
+];
+
+export const borders = [
+  [/^border$/, handlerBorder],
+  [/^border()-(\d+)$/, handlerBorder, { autocomplete: "(border)-<directions>" }],
+  [/^border-([xy])-(\d+)$/, handlerBorder],
+  [/^border-([rltb])$/, handlerBorder],
+  [/^border-([rltb])-(\d+)$/, handlerBorder],
+  [/^border()-(.+)$/, handlerBorderStyle, { autocomplete: "(border)-style" }],
+  [/^border-([xy])-(.+)$/, handlerBorderStyle],
+  [/^border-([rltb])-(.+)$/, handlerBorderStyle],
+];
+
+function handlerBorder(m, ctx) {
+  const borderSizes = handlerBorderSize(m, ctx);
+  const borderStyle = handlerBorderStyle(["", m[1], "solid"]);
```

Comment:
I think `handlerBorderStyle` might need tweaked or removed here.

Since we inherit [Tailwind's reset](https://github.com/warp-ds/drive/blob/master/plugin/src/_preflights/tw-reset.js#L14) we already have all borders set to be 'solid' - so we don't really want the 'bloat' on (f.ex) a `border-b-2` rule to include `border-style: solid`.